features/frameworks/mbed-trace/MbedTraceInitialiser.cpp

```diff
@@ -0,0 +1,12 @@
+#include "mbed_trace.h"
+
+class MbedTraceInitialiser {
+public:
+    MbedTraceInitialiser() {
+        mbed_trace_init();
+    }
+};
+
+#if MBED_CONF_MBED_TRACE_ENABLE
+MbedTraceInitialiser initialiser;
```

Comment:
extern symbol - not private for just this implementation (anonymous namespace) ?